04-Custom-Actions-In-Python/02-An-Initial-Custom-Action.ipynb

```diff
@@ -109,7 +109,7 @@
     "custom action is, we must wrap it in a subclass of\n",
     "`hoomd.custom.CustomOperation`. We have three options as discussed in\n",
     "the previous section: an updater, writer, or tuner. Since our object\n",
-    "does not modify simulation state or an object's hyperparameters, but\n",
+    "does not modify the simulation state or an object's hyperparameters, but\n",
```

Comment:
```suggestion
    "does not modify the simulation state or an object's attributes, but\n",
```
I pushed a commit that removes the other invalid uses of the word "hyperparameter".